bibliopixel/project/fix.py

```diff
@@ -0,0 +1,92 @@
+import copy
```

Comment:
Why is this called fix?

Reply:
Uh, good question.  :-)

The idea is that they "fix" up a project description.

Not a great name izzit?  Ideas?